### ISSUE TITLE
Error after updating React-Native-Windows , React-Native and React

### ISSUE THREAD

**Issue by paulbx81** [Microsoft/react-native-windows#2025]
### Environment

1. `react-native -v`: 0.57.5
2. `npm ls rnpm-plugin-windows`: 0.2.8
3. `npm ls react-native-windows`: 0.57.0-rc.0
4. `node -v`: 11.2.0
5. `npm -v`: 6.4.1
6. `react`: 16.6.3

Then, specify:

- Target Platform: UWP

- Target Platform Version(s): 10.0.17134

- Target Device(s): Computer windows 10 x64

- Development Operating System: windows 10 pro

- Visual Studio Version:

- Build Configuration: Debug

### Steps to Reproduce

I was in : 
- react-native-windows 0.55.0
- React-native 0.55.4
- React: 16.3.1

And i decided to upgrade all of this to the latest version

### Actual Behavior

![rip](https://user-images.githubusercontent.com/40029591/48766507-ee3c5580-ecb4-11e8-9355-bdd5e7e6bab1.PNG)

Thx for your help !

**Comment by rozele** [comment]
@paulbx81 - does this still repro with RN 0.57.8?

**Comment by phorensic** [comment]
I broke my environment trying to upgrade react-native and react. I have found that the newest versions of everything that work are:
"react": "16.5.0",
"react-native": "^0.57.0",
"react-native-windows": "^0.57.0-rc.5"

There is also a breaking change in babel, so that if you are creating a new project the babel folders are not installed properly. In this case I needed to "npm install --save-dev @babel/runtime".

It seems react-native-windows is lagging behind in development. So as react-native pushes forward you cannot test out the new stuff.
